app/Repositories/Eloquents/BookEloquentRepository.php

```diff
@@ -289,4 +289,15 @@ public function countBook()
     {
         return $this->model()->all()->count();
     }
+
+    public function updateBook($id, $data)
+    {
+        $book = $this->model()->find($id);
+        $description = strip_tags($data['description'], config('view.text'));
+        $data['description'] = preg_replace('/(<[^>]+) style=".*?"/i', '$1', $description);
+        $book->update($data);
+
+        return $book;
+
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>**Line**: 302</br>Function closing brace must go on the next line following the body; found 1 blank lines before brace